components/k8s-model-server/images/Dockerfile.cpu

```diff
@@ -37,17 +37,8 @@ RUN apt-get update && apt-get install -y \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN echo "deb [arch=amd64] http://storage.googleapis.com/tensorflow-serving-apt stable \
- tensorflow-model-server \
- tensorflow-model-server-universal" \
- | tee /etc/apt/sources.list.d/tensorflow-serving.list && \
- curl https://storage.googleapis.com/tensorflow-serving-apt/tensorflow-serving.release.pub.gpg \
- | apt-key add -
-
-RUN apt-get update && apt-get install -y \
-        tensorflow-model-server && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+RUN curl -O http://storage.googleapis.com/tensorflow-serving-apt/pool/tensorflow-model-server/t/tensorflow-model-server/tensorflow-model-server_1.4.0_all.deb
+RUN dpkg -i tensorflow-model-server_1.4.0_all.deb
```

Comment:
Do we also build 1.4 for GPU?

Reply:
yes https://github.com/kubeflow/kubeflow/blob/master/components/k8s-model-server/images/Dockerfile.gpu#L76